src/learn/DOTNET/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Introduction to Dotnet
+description: Learn how to set up a basic DOTNET app
+tags:
+  - workshop
+keywords:
+  - DOTNET
+  - ASP.NET 
+  - CSharp
+challenge: https://github.com/foundersandcoders/DOTNET-Workshop
+---
+
+Although all are coding so far has been in javascript or its close cousin typescript not all web sites are coded in this language. If you know other popular coding languges it will boost you capabilities as a dev.
```

Comment:
```suggestion
Although all our coding so far has been in javascript or its close cousin typescript not all web sites are coded in this language. If you know other popular coding languages it will boost you capabilities as a dev.
```